tools/remove_dependencies.sh

```diff
@@ -25,107 +25,107 @@ function fail() {
 
 BUILD_JAVA=${BUILD_JAVA:-1}
 BUILD_CONSUL=${BUILD_CONSUL:-1}
-uname=$(uname)
-get_arch=$(uname -m)
+UNAME=$(uname)
+ARCH=$(uname -m)
 
 uninstall_protoc() {
     echo "Removing protoc..."
-    local dist="$1"
+    local DIST="$1"
 
-    if [ -f "$dist/bin/protoc" ]; then
-        unlink "$dist/bin/protoc"
+    if [ -f "$DIST/bin/protoc" ]; then
+        unlink "$DIST/bin/protoc"
         rm "$VTROOT/bin/protoc"
     fi
-    if [[ "$(echo $dist | awk -F/ '{print $NF}')" == "vt-protoc-$PROTOC_VER" ]]; then
-        rm -rf $dist
+    if [[ "$(echo $DIST | awk -F/ '{print $NF}')" == "vt-protoc-$PROTOC_VER" ]]; then
```

Comment:
I saw your previous commit that changed these but now don't see them here for some reason.